### PR TITLE
404_template added

### PR DESCRIPTION
**Pull request by prihit**

### CODE REVIEW

File: profiles/views/authentication.py [hunk 1]
```diff
@@ -51,10 +51,8 @@ def post(self, request):
             'username': username,
             'first_name': first_name,
         }
-        try:
-            generic_mailer(**context)
-        except:
-            pass
+        import ipdb; ipdb.set_trace()
```
Comment (chayan007) [inline profiles/views/authentication.py:54]: remove ipdb @prihit

File: reviews/migrations/0002_auto_20210527_1945.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.1.5 on 2021-05-27 14:15
```
Comment (chayan007) [inline reviews/migrations/0002_auto_20210527_1945.py:1]: remove this migration

File: profiles/views/authentication.py (hunk 1 above)
Comment (prihit) [reply profiles/views/authentication.py]: done

File: reviews/migrations/0002_auto_20210527_1945.py (hunk 2 above)
Comment (prihit) [reply reviews/migrations/0002_auto_20210527_1945.py:1]: Done

**Pull request merged**